.github/workflows/ci.yml

```diff
@@ -9,6 +9,7 @@ on:
       # branches. Without this we would end up running this job twice for non
       # forked PRs, once for the push and then once for opening the PR.
       - "**:**"
+      - master
```

Comment:
This should be reverted.

Reply:
👌🏻

Reply:
Github workflow doesn't trigger on a pull request for the master branch

Reply:
But shouldn't it trigger on `push` to the forked branch?